modules/ROOT/pages/munit-extensions-maven-plugin-configuration.adoc

```diff
@@ -77,6 +77,28 @@ line to the runtime configuration:
 
 From the command line, the property is `discoverRuntimes.includeSnapshots`.
 
+Default value is `false`.
+
+=== To Discover only latest patches
+
+If you want to discover only the latest patch versions of the runtime you can add the following
```

Comment:
the runtime > Mule runtime engine, or Mule

Reply:
Or does MUnit have its own runtime? Or do you mean the MUnit configuration for Mule? or...?